### DISCUSSION TITLE
Does the DietPi image for Odroid N2+ support panfrost drivers?

### DISCUSSION THREAD

**Discussion by kartikynwa** [MichaIng/DietPi#6494] [Q&A] [answered]
Hello.

Extremely new to the SBC game. I have an Odroid N2+ that I use mostly as a Jellyfin server.

Does the DietPi image for this board have support for the panfrost GPU drivers?

Thanks.

**Comment by MichaIng** [comment, marked as answer]
Sorry for the late reply.

I have not tested it recently, but the kernel ships with the `panfrost` kernel module and the Debian Mesa drivers support this generally well. And the SBC is sufficiently old to expect good mainline kernel and Mesa support.
